src/lib.rs

```diff
@@ -1,7 +1,52 @@
 // Copyright © 2019 Intel Corporation. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0 OR BSD-3-Clause
 
-//! rust-vmm device model.
```

Comment:
I think we can also add the `#![deny(missing_docs)]` now. It also looks like [this public method](https://github.com/rust-vmm/vm-device/blob/main/src/bus/range.rs#L44) is not documented yet.

Reply:
Good catch!